core/keys/symbols.py

```diff
@@ -0,0 +1,122 @@
+# define the spoken forms for symbols in command and dictation mode
```

Comment:
This file now became quite complicated. We're leaning towards just having one list with the constants/definitions and then a separate file with all the logic.

Reply:
I have small preference for keeping this as-is, so everything can be customized in one place. We can discuss further in the next session.